### ISSUE TITLE
Amazon gift card SCAM - NEED RESPONSE FROM SOMEBODY AT BITPAY

### ISSUE THREAD

**Issue by altafx** [bitpay/copay#7590]
TLDR - Purchased an amazon gift card worth almost $1,000. BitPay says issuing the card to my account failed, but they took my bitcoin anyway. Now I have no bitcoin, no gift card, and **absolutely no response from anybody at BitPay for over 10 days.** [Old Issue](https://github.com/bitpay/copay/issues/7578)

I've posted multiple tickets here, all of which get closed with no explanation or response other than "contact support". Well I HAVE CONTACTED SUPPORT AND CAN'T GET AN ANSWER, WHICH IS WHY I AM POSTING HERE PUBLICLY. I NEED A DAMN RESPONSE FROM BITPAY ON THIS SITUATION

Closing my tickets and telling me to email BitPay support (which is a black hole where emails go but responses never come back) is NOT an acceptable solution to this problem.

**Comment by JDonadio** [comment]
I already explain the procedure to report your issue, unfortunately from here we could not do much.
